### ISSUE TITLE
5.0.6: pytest warnings

### ISSUE THREAD

**Issue by kloczek** [DiffSK/configobj#214]
Looks like latest pytest shows some warnings
```console
============================================================================= warnings summary =============================================================================
tests/test_configobj.py:41
  /home/tkloczko/rpmbuild/BUILD/configobj-5.0.6/tests/test_configobj.py:41: DeprecationWarning: invalid escape sequence \s
    ws_chars = len(re.search('^(\s*)', first_content).group(1))

tests/test_configobj.py:51
  /home/tkloczko/rpmbuild/BUILD/configobj-5.0.6/tests/test_configobj.py:51: DeprecationWarning: invalid escape sequence \s
    return [re.sub('^\s{0,%s}' % ws_chars, '', line).encode('utf-8')

-- Docs: https://docs.pytest.org/en/stable/warnings.html
====================================================================== 71 passed, 2 warnings in 0.38s ======================================================================
```

**Comment by robdennis** [comment]
if I remember correctly, warnings are going to be python-version specific. If you're able (it's been a year :( ) add here what python version this is with.

Thanks

**Comment by kloczek** [comment]
Sorry I shout submit full pytest output 😋 
Python 3.8.15.

**Comment by robdennis** [comment]
this is also showing in CI/CD
https://github.com/DiffSK/configobj/actions/runs/3943624839/jobs/6748667913#step:5:27

**Comment by jelmer** [comment]
Both of these are fixed in 5.0.x and master.
